Blaise-Shyaka/politico-react
# [#172014390] Create sign up page

# What does this PR do?

It creates a page allowing users to create an account

# Description of the tasks to be completed

- Create a page with a form to collect a user's personal information
- The form should contain at least email and phone number inputs
- Make the page simple and responsive

# How should this be manually tested?

- Clone this repository
- In the root directory run `npm start` to view the app in your default browser
- A home page should show up
- Click on 'signup' or 'get started' buttons
- A page with a form should show up
- Resize your browser to verify its responsiveness

# What are the relevant pivotal tracker stories?

#172014390

# Screenshots

![Screenshot (187)](https://user-images.githubusercontent.com/52489421/77992634-bd4b4680-7326-11ea-9c86-75ffbe8ca4f8.png)
![Screenshot (188)](https://user-images.githubusercontent.com/52489421/77992637-bde3dd00-7326-11ea-9098-fa18c3d54818.png)
![Screenshot (189)](https://user-images.githubusercontent.com/52489421/77992635-bde3dd00-7326-11ea-8412-fbacee276841.png)
